package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dellstorage/clarity-react",
-    "version": "1.2.9",
+    "version": "1.2.11",
```

Comment:
```suggestion
    "version": "1.2.10",
```

Reply:
Hi @akate1 , package.json in objectscale-portal is 1.2.10, should I still change this to 1.2.10?

Reply:
yes. the npm package version for clarity will be 1.2.11